### PR TITLE
Feature/nutrition game

### PR DESCRIPTION
**Pull request by kingam280**

### CODE REVIEW

**Review by mateugazi — CHANGES_REQUESTED** [review]
Were you using parcel when you worked on that feature? I don't see that file in "package.json" -> "scripts" -> "start:dev" and in "build" (but that we can add at the end, when we finished the merging). When I add it to the parcel the game couldn't load, because of "Uncaught ReferenceError: regeneratorRuntime is not defined" . We are working on this project with parcel, so I think that should be fixed. I already fixed that in my local repo, but I also don't want to do something on your branch without your allowance, so I'm waiting for your answer

**Comment by kingam280** [comment]
I was working on live server from VS Code but it's fixed and should work now.

**Comment by mateugazi** [comment]
Yes it works now, thanks.

File: src/app/Game.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        this.imgInput.src = await currentQuestion.imgSrc
+        this.imgInput.alt = await currentQuestion.name
+        this.dishNameInput.textContent = await currentQuestion.name
+        timer.startTimer(await question, this)
+        
+        this.sumUpRoundFunction = this.sumUpRound.bind(this, question, timer)
+        this.checkBtn.addEventListener("click", this.sumUpRoundFunction, {once: true})  
+    }
+
+    sumUpRound(question, timer, e) {
```
Comment (mateugazi) [inline src/app/Game.js:58]: Radek said that we should avoid one letter variables, even when they are obvious

File: src/app/Game.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        this.answerInput = document.querySelector(".answer")
+        this.correctAnswer = document.querySelector(".correct-answer")
+        this.gameOverBlock = document.querySelector(".game-over")
+        this.finalScoreInput = document.querySelector(".game-over .score span")
+
+        this.stats = new Stats(0)
+        
+        document.addEventListener('DOMContentLoaded', () => {
+            this.startBtn.addEventListener("click", this.startRound.bind(this), {once: true})
+            this.checkBtn.addEventListener("click", (e) => e.preventDefault())
```
Comment (mateugazi) [inline src/app/Game.js:34]: I will copy and paste previous comment: "Radek said that we should avoid one letter variables, even when they are obvious "

**Review by mateugazi — DISMISSED** [review]
To me it's looks cool (I personally don't think that one name variable is a issue here) so you have my approval and btw great job :)

**Review by MaDud — DISMISSED** [review]

File: src/app/Question.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        this.currentQuestion;
+        this.questions = questions
+    }
+
+    getQuestion() {
+        const index = Math.floor(Math.random() * this.questions.length)
+        this.currentQuestion = this.questions[index];
+        this.questions.splice(index, 1);
+        return fetch(`https://api.spoonacular.com/recipes/guessNutrition?title=${this.currentQuestion.apiTitle}&apiKey=08dba6e965974fdb9c6a8cc7b0f8f4f0`)
+            .then( res => res.json() )
```
[the remark this answers is not in the document]
Comment (kingam280) [reply src/app/Question.js:12]: Okay, I'll add if statement there

**Review by mateugazi — APPROVED** [review]

**Review by mateugazi — DISMISSED** [review]

**Review by MaDud — APPROVED** [review]

**Review by mateugazi — APPROVED** [review]

**Pull request merged**